shared/common-adapters/floating-box.desktop.js

```diff
@@ -9,26 +9,41 @@ const StyleOnlyBox = (props: any) => <Box children={props.children} />
 const RelativeFloatingBox = ModalPositionRelative(StyleOnlyBox)
 
 type State = {
-  targetRect: ?ClientRect,
+  targetRect: ?React.Component<any, any>,
```

Comment:
I get a flow error in render:
```
Cannot create RelativeFloatingBox element because React.Component [1] is incompatible with ClientRect [2] in property targetRect.
```

Reply:
fixed